crates/sui-keys/src/random_names.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::{rngs::ThreadRng, thread_rng, Rng};
+
+/// Return a random name formatted as first-second from a list of strings.
+///
+/// The main purpose of this function is to generate random aliases for addresses.
+pub fn random_name(conflicts: Vec<String>) -> String {
+    let left_names = vec![
```

Comment:
These lists can be `const LEFT_NAMES: &[&str] = &[ ... ]` etc, to avoid reallocating them on every call.